optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/statistic/bestscore/BestScoreProblemStatistic.java

```diff
@@ -73,14 +73,16 @@ public SubSingleStatistic createSubSingleStatistic(SubSingleBenchmarkResult subS
     public void writeGraphFiles(BenchmarkReport benchmarkReport) {
         List<XYPlot> plotList = new ArrayList<XYPlot>(BenchmarkReport.CHARTED_SCORE_LEVEL_SIZE);
         int seriesIndex = 0;
-        for (SingleBenchmarkResult singleBenchmarkResult : problemBenchmarkResult.getSingleBenchmarkResultList()) {
+        List<SingleBenchmarkResult> results = problemBenchmarkResult.getSingleBenchmarkResultList();
+        for (SingleBenchmarkResult singleBenchmarkResult : results) {
```

Comment:
Is there any advantage to writing for-each loops like this, instead of the original version?

Also, we usually name lists ...List, f.e. "pointList" (an array would be "points") IIRC.